sw/device/silicon_creator/lib/cert/dice_cwt.c

```diff
@@ -149,6 +363,8 @@ rom_error_t dice_cdi_1_cert_build(hmac_digest_t *owner_measurement,
                                   cert_key_id_pair_t *key_ids,
                                   ecdsa_p256_public_key_t *cdi_1_pubkey,
                                   uint8_t *cert, size_t *cert_size) {
-  // TODO(lowRISC/opentitan:#24281): implement body
+  HARDENED_RETURN_IF_ERROR(
+      dice_cdi_0_cert_build(owner_measurement, owner_security_version, key_ids,
```

Comment:
The `owner_manifest_measurement` is missing from the cert. This contains the measurement of the "owner configuration block", should this be included?

Reply:
Yep, it can be part of the "Authority Descriptor"